src/registry.ts

```diff
@@ -4,28 +4,29 @@ import type { IntercomInput } from './runtime/registry/intercom'
 import type { SegmentInput } from './runtime/registry/segment'
 import type { NpmInput } from './runtime/registry/npm'
 import type { PlausibleAnalyticsInput } from './runtime/registry/plausible-analytics'
-import type { GoogleAnalyticsInput } from './runtime/registry/google-analytics'
+// import type { GoogleAnalyticsInput } from './runtime/registry/google-analytics'
 import type { RegistryScripts } from './runtime/types'
 import type { GoogleAdsenseInput } from './runtime/registry/google-adsense'
 
 // avoid nuxt/kit dependency here so we can use in docs
 
 export const registry: (resolve?: (s: string) => string) => RegistryScripts = (resolve?: (s: string) => string) => {
   resolve = resolve || ((s: string) => s)
+
   return [
     // analytics
-    {
-      label: 'Google Analytics',
-      category: 'analytics',
-      logo: `<svg xmlns="http://www.w3.org/2000/svg" width="28.85" height="32" viewBox="0 0 256 284"><path fill="#F9AB00" d="M256.003 247.933a35.224 35.224 0 0 1-39.376 35.161c-18.044-2.67-31.266-18.371-30.826-36.606V36.845C185.365 18.591 198.62 2.881 216.687.24a35.221 35.221 0 0 1 39.316 35.16z"/><path fill="#E37400" d="M35.101 213.193c19.386 0 35.101 15.716 35.101 35.101c0 19.386-15.715 35.101-35.101 35.101S0 267.68 0 248.295c0-19.386 15.715-35.102 35.101-35.102m92.358-106.387c-19.477 1.068-34.59 17.406-34.137 36.908v94.285c0 25.588 11.259 41.122 27.755 44.433a35.161 35.161 0 0 0 42.146-34.56V142.089a35.222 35.222 0 0 0-35.764-35.282"/></svg>`,
-      scriptBundling: (options?: GoogleAnalyticsInput) => withQuery('https://www.googletagmanager.com/gtag/js', {
-        id: options?.id,
-      }),
-      import: {
-        name: 'useScriptGoogleAnalytics',
-        from: resolve('./runtime/registry/google-analytics'),
-      },
-    },
+    // {
+    //   label: 'Google Analytics',
+    //   category: 'analytics',
+    //   logo: `<svg xmlns="http://www.w3.org/2000/svg" width="28.85" height="32" viewBox="0 0 256 284"><path fill="#F9AB00" d="M256.003 247.933a35.224 35.224 0 0 1-39.376 35.161c-18.044-2.67-31.266-18.371-30.826-36.606V36.845C185.365 18.591 198.62 2.881 216.687.24a35.221 35.221 0 0 1 39.316 35.16z"/><path fill="#E37400" d="M35.101 213.193c19.386 0 35.101 15.716 35.101 35.101c0 19.386-15.715 35.101-35.101 35.101S0 267.68 0 248.295c0-19.386 15.715-35.102 35.101-35.102m92.358-106.387c-19.477 1.068-34.59 17.406-34.137 36.908v94.285c0 25.588 11.259 41.122 27.755 44.433a35.161 35.161 0 0 0 42.146-34.56V142.089a35.222 35.222 0 0 0-35.764-35.282"/></svg>`,
```

Comment:
general question: what is the svg logo used for? 
It doesn't have any functional purpose that I can think of.

Reply:
Nuxt Devtools and the docs, it shouldn't get bundled into runtime code